synfig-studio/src/gui/instance.cpp

```diff
@@ -1766,7 +1786,7 @@ Instance::add_special_layer_actions_to_group(const Glib::RefPtr<Gtk::ActionGroup
 	{
 		String action_name = etl::strprintf("special-action-open-file-%d", index);
 		//if the import layer is type image 
-		if(std::find(img_ext.begin(), img_ext.end(), filename_extension(i->second)) != img_ext.end())
+		if(is_img(filename_extension(i->second)))
```

Comment:
This makes possible to remove the following line above (at the beginning of add_special_layer_actions_to_group() function) ^__^ -
```vector <String> img_ext{".jpg",".jpeg",".png",".bmp",".gif"};```

Reply:
Ah it's embarrassing